facetapi.install

```diff
@@ -73,30 +73,36 @@ function facetapi_schema() {
     ),
     'primary key' => array('name'),
   );
-
+  $schema['facetapi_config'] = array(
```

Comment:
I'm still not understanding why youre creating this table instead of storing these variables in CMI.